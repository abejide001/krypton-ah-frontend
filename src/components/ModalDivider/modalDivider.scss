@import "../../styles/partials/colors";
@import "../../styles/partials/fonts";

.ui {
  &.horizontal {
    &.divider {
      font: {
        size: 0.875rem;
        weight: 100;
      };
      text-transform: none;
    }
  }
}
