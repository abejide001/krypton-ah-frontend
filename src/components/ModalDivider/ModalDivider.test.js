import React from 'react';
import { shallow } from 'enzyme';
import ModalDivider from '.';

describe('SignIn Form', () => {
  test('should matches the snapshot', () => {
    const wrapper = shallow(<ModalDivider />);
    expect(wrapper).toMatchSnapshot();
  });
});
