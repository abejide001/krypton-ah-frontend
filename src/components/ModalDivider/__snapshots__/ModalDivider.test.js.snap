// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignIn Form should matches the snapshot 1`] = `
<Divider
  horizontal={true}
>
  sign in
</Divider>
`;
